submodules/lev/lev/src/lev_stubs.c

```diff
@@ -57,9 +57,6 @@
     CAMLreturn(Val_long(__value));                                             \
   }
 
-CAMLextern int caml_convert_signal_number(int);
-CAMLextern int caml_rev_convert_signal_number(int);
```

Comment:
This commit can stay. We just need it in lev as well.

Reply:
remove the revert of that particular commit; here's the PR to lev: https://github.com/rgrinberg/lev/pull/86